src/Widget/CommandHandler/MigrateWidgetPageCommandHandler.php

```diff
@@ -200,6 +200,8 @@ protected function serializeWidgetPage($data, $project)
             // Combine and set CSS.
             $cssMigration = new CssMigration($data['blocks']);
             $widgetPageEntity->setCss($cssMigration->getCss());
+            // set the selected theme
```

Comment:
This comment is literally the same as the name of the method it is commenting on. It brings no extra information so I would say it's not needed.